Automattic/jetpack
# Tiled Gallery: fix view rendering issues

Fixes https://github.com/Automattic/jetpack/issues/37194

## Proposed changes:

This PR fixes the Tiled Gallery not being rendered as expected in the frontend. The issues was introduced by https://github.com/Automattic/jetpack/pull/36854.

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Don't checkout this branch just yet
- Spin up a test site
- Connect Jetpack
- Create a new post
- Add a _Tiled Gallery_ block and select some images
- Checkout this branch
- Refresh the editor and verify there's no error message
- In the block styles panel, select _Circles_
<img width="276" alt="Screenshot 2024-05-03 at 10 56 47 AM" src="https://github.com/Automattic/jetpack/assets/1620183/22a71db9-ddd4-4462-a98d-9bd7df8eec90">

- Open the preview
- Notice the items are circular
<img width="500" alt="Screenshot 2024-05-03 at 10 54 27 AM" src="https://github.com/Automattic/jetpack/assets/1620183/e717acfb-6606-4eba-a22d-4032f968971a">

- Verify the other styles are rendered the same in the editor and frontend
